src/service/templates/service.filter.js

```diff
@@ -1,13 +1,17 @@
-module.exports = function() {
-  if(process.env.NODE_ENV !== 'production') {
-    console.warn(`You are using the default filter for the \`{{options.name}}\` service (in ${__dirname}).`);
-    console.warn(`This means all clients will get every real-time event.`);
-    console.warn(`For more information how to filter events see http://docs.feathersjs.com/real-time/filtering.html`);
-    console.warn(``);
+module.exports = function () {
+  function log () {
+    if (process.env.NODE_ENV !== 'testing') {
+      console.log(...arguments);
+    }
   }
 
-  return function() {
-    console.log(`{{options.name}} filter returned: true`);
+  log(`You are using the default filter for the \`{{options.name}}\` service (in ${__dirname}).`);
+  log(`This means all clients will get every real-time event.`);
+  log(`For more information how to filter events see http://docs.feathersjs.com/real-time/filtering.html`);
+  log(``);
+
+  return function () {
+    log(`{{options.name}} filter returned: true`);
```

Comment:
@ekryski can you point me at the new proposal? Happy to update the filtering to be more secure, that's definitely the best practice, I just want to make sure that when generations occur, it takes nothing more then an `npm start` to see it work. Right now the logging that occurs on `npm start` to warn you that you are using defaults is what we're using to make sure that users are aware of this default.